app/controllers/application_drafts_controller.rb

```diff
@@ -81,8 +81,12 @@ def application_draft
   end
 
   def application_draft_params
-    params.require(:application_draft).
-      permit(:project1_id, :project2_id, :project_plan, :misc_info, :heard_about_it, :voluntary, :voluntary_hours_per_week, :working_together, :why_selected_project)
+    params.require(:application_draft).permit(
+      :project1_id, :project2_id,
+      :plan_project1, :plan_project2,
+      :why_selected_project1, :why_selected_project2,
+      :misc_info, :heard_about_it, :voluntary, :voluntary_hours_per_week, :working_together
```

Comment:
So just took a look at it again - seems like it covers all attributes from the form, so missing the rest should be on purpose ✌️ 
*(btw: wifi and "beinfreiheit" on ICE is awesome 🎉  did not know about this)*